composer.json

```diff
@@ -7,10 +7,11 @@
     "prefer-stable": true,
     "replace": {
         "symfony/polyfill-php72": "*",
-        "symfony/polyfill-php73": "*"
+        "symfony/polyfill-php73": "*",
+        "symfony/polyfill-php74": "*"
```

Comment:
you can also add `symfony/polyfill-php80` here

Reply:
Yes! Done. Thanks.